### PR TITLE
Adds new station traits that gives departments / everyone death rattle implants

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
add: Added new station traits that gives departments / everyone death rattle implants
/:cl:

seems fun and innocent enough.

### CODE REVIEW

**Comment by maxymax13** [comment]
curator, assistant, and chaplain are now "DEPARTMENT_SERVICE". civilian is officially no longer real

**Comment by maxymax13** [comment]
actually make assistants be a civilian department so that assistants don't benefit from the benefits of hard working service jobs and vice versa

File: code/modules/jobs/job_types/psychologist.dm [hunk 1]
```diff
@@ -17,6 +17,7 @@
 	liver_traits = list(TRAIT_MEDICAL_METABOLISM)
 
 	display_order = JOB_DISPLAY_ORDER_PSYCHOLOGIST
+	departments = DEPARTMENT_MEDICAL
```
Comment (Fikou) [inline code/modules/jobs/job_types/psychologist.dm:20]:
```suggestion
	departments = DEPARTMENT_SERVICE
```
hes a service job

Comment (Qustinnus) [reply code/modules/jobs/job_types/psychologist.dm:20]: is he though

**Comment by cacogen** [comment]
This is as though a means of testing a desired new feature while avoiding the inevitable backlash. Anyway, for members of departments to know when other members have died is an interesting balance change and I'd like to see how it plays out on the rounds this shows up in.

File: code/modules/jobs/job_types/psychologist.dm (hunk 1 above)
Comment (RaveRadbury) [reply code/modules/jobs/job_types/psychologist.dm:20]: He is, original PR says so

**Comment by Mothblocks** [comment]
image

**Comment by Mothblocks** [comment]
test your CODE

**Comment by Qustinnus** [comment]
I mean I would but the pr isn't done so kind of pointless

**Pull request merged**